suite-native/transactions/src/components/transactionDetail/TransactionDetailAmount.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
nit: I would prefer to have these folders in components (I mean `transactionDetail` and `transactionsList` ) named with camel case as we have this in our **atoms** package...

![Snímek obrazovky 2022-09-15 v 17 33 16](https://user-images.githubusercontent.com/2227592/190446013-1ea4a4c5-60fd-4556-bb6e-aef8d2269b70.png)